zulipterminal/config/themes.py

```diff
@@ -29,6 +29,7 @@
     'reaction_mine': 'standout',
     'msg_mention': 'bold',
     'msg_link': '',
+    'msg_link_index': 'bold',
```

Comment:
While the previous commit is fine first, if you have this change first, you don't need to change all the uses of it to be the new names.

Reply:
True. Notwithstanding, the changes in this particular commit express where the new style is used explicitly [in GitHub's diff].

Reply:
I agree that the ordering you have is reasonable, it just ends up with a larger diff.

Reply:
Yeah, that's there. However, it only fits with the previous commit which is pretty large already.